grpc/service_config/service_config.proto

```diff
@@ -412,6 +415,7 @@ message XdsServer {
 }
 
 // Configuration for xds_cluster_resolver LB policy.
+// Deprecated.
 message XdsClusterResolverLoadBalancingPolicyConfig {
```

Comment:
Optional: It might be nice to move these deprecated proto messages to another file so they don't clutter up the non-deprecated ones.  E.g. `deprecated.proto`?

Reply:
I've moved the deprecated messages to a separate section at the end of the file.